spec/controllers/events_controller_spec.rb

```diff
@@ -105,7 +105,7 @@
       let(:params) { FactoryGirl.attributes_for(:event) }
 
       it 'throws a not authorized error' do
-        expect { subject }.to raise_error(Pundit::NotAuthorizedError)
+        expect(subject).to redirect_to forbidden_path
```

Comment:
@brendandeere & @jacquesporveau, after updating the approach used to display the error page, the `Pundit::NotAuthorizedError` is raised and can be tested. 

Agree with @graygilmore that the shared example approach would be a great way to DRY up our specs. Since this pattern was established earlier in the project, I think this could be a separate chore on our pivotal board, rather than something that gets changed for this PR. I have added to pivotal icebox!